cmd/import.go

```diff
@@ -338,7 +338,7 @@ func importUsers(ctx context.Context, provider pager.Pager, fh *firehydrant.Clie
 		switch selected {
 		case 0:
 			console.Infof("[+] All users will be created in FireHydrant.\n")
-			for _, u := range unmatched[i:] {
+			for _, u := range toImport[i:] {
```

Comment:
this bug caused us to provision _everyone_ regardless of the people who were selected 🙃 i'm going to follow up with a refactor + test. this is a quick fix for now.